pkg/testing/v1alpha1/revision.go

```diff
@@ -153,3 +153,13 @@ func MarkRevisionReady(r *v1alpha1.Revision) {
 	r.Status.MarkResourcesAvailable()
 	r.Status.MarkContainerHealthy()
 }
+
+// WithRevisionLabel attaches a particular label to the revision.
+func WithRevisionLabel(key, value string) RevisionOption {
```

Comment:
nit: It would be nice to rename this to avoid ambiguity. `{xyz}Label` is used frequently in the code base to describe the `serving.knative.dev/{xyz}` key/value pair while this is describing adding an arbitrary key/value to a Revision object.

```suggestion
func LabelRevisionWith(key, value string) RevisionOption {
```

Reply:
This is actually consistent with `With{xyz}Label` functions IIRC, so perhaps we should defer and rename all?